app/src/main/java/com/example/recipeopedia/activities/EditProfileActivity.java

```diff
@@ -0,0 +1,101 @@
+package com.example.recipeopedia.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.example.recipeopedia.R;
+import com.parse.ParseUser;
+
+public class EditProfileActivity extends AppCompatActivity {
+    public static final String TAG = "EditProfileActivity";
+    private EditText etFirstName, etLastName, etEmail, etPhoneNumber, etBio;
```

Comment:
Similar to this, you can also create String variables for each of these EditText variables which means you'd only have to call getText().toString() once per variable